lib/admin/accounts.ex

```diff
@@ -352,4 +353,35 @@ defmodule Admin.Accounts do
       confirmed: Admin.Repo.aggregate(from(u in User, where: not is_nil(u.confirmed_at)), :count)
     }
   end
+
+  # Graasp Members
+
+  def get_member!(id) do
+    Repo.get!(Account, id)
+  end
+
+  def get_member_by_email(email) do
+    case Repo.get_by(Account, email: email) do
+      %Account{} = user -> {:ok, user}
+      nil -> {:error, :member_not_found}
+    end
+  end
+
+  def get_active_members do
+    Repo.all(
+      from(m in Account,
+        where:
+          not is_nil(m.last_authenticated_at) and m.last_authenticated_at > ago(90, "day") and
+            m.type == "individual",
```

Comment:
Question: the "type" of `type` is string right? Is there any safe in case of typo?